locales/es/es.json

```diff
@@ -644,67 +645,67 @@
     "You have been invited to join the :team team!": "¡Usted ha sido invitado a unirse al equipo :team!",
     "You have enabled two factor authentication.": "Ha habilitado la autenticación de dos factores.",
     "You have not enabled two factor authentication.": "No ha habilitado la autenticación de dos factores.",
-    "You may cancel your subscription at any time. Once your subscription has been cancelled, you will have the option to resume the subscription until the end of your current billing cycle.": "You may cancel your subscription at any time. Once your subscription has been cancelled, you will have the option to resume the subscription until the end of your current billing cycle.",
+    "You may cancel your subscription at any time. Once your subscription has been cancelled, you will have the option to resume the subscription until the end of your current billing cycle.": "Puedes cancelar tu subscripción en cualquier momento. Una vez que su suscripción haya sido cancelada, tendrá la opción de reanudar la suscripción hasta el final de su ciclo de facturación actual.",
     "You may delete any of your existing tokens if they are no longer needed.": "Puede eliminar cualquiera de sus tokens existentes si ya no los necesita.",
     "You may not delete your personal team.": "No se puede borrar su equipo personal.",
     "You may not leave a team that you created.": "No se puede abandonar un equipo que usted creó.",
-    "Your :invoiceName invoice is now available!": "Your :invoiceName invoice is now available!",
-    "Your card was declined. Please contact your card issuer for more information.": "Your card was declined. Please contact your card issuer for more information.",
-    "Your current payment method is a credit card ending in :lastFour that expires on :expiration.": "Your current payment method is a credit card ending in :lastFour that expires on :expiration.",
+    "Your card was declined. Please contact your card issuer for more information.": "Su tarjeta fue rechazada. Comuníquese con el emisor de su tarjeta para obtener más información.",
+    "Your current payment method is a credit card ending in :lastFour that expires on :expiration.": "Su método de pago actual es una tarjeta de crédito que termina en :lastFour que vence el :expiration.",
     "Your email address is not verified.": "Su dirección de correo electrónico no está verificada.",
-    "Your registered VAT Number is :vatNumber.": "Your registered VAT Number is :vatNumber.",
+    "Your registered VAT Number is :vatNumber.": "Su número VAT registrado es :vatNumber.",
+    "Your :invoiceName invoice is now available!": "¡Su factura :invoiceName ya está disponible!",
     "Zambia": "Zambia",
     "Zimbabwe": "Zimbabwe",
-    "Zip \/ Postal Code": "Zip \/ Postal Code"
+    "Zip / Postal Code": "Zip / Código postal"
```

Comment:
need "\\"

Reply:
Problems in [`source/en.json`](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L709). There is the same problem

Reply:
I will fix source/en.json